File: ppg_sl/php/enjou.php
//言語：php
//問題URL:https://paiza.jp/works/mondai/ppg_sl_php/ppg_sl_php__enjou
//コメント：
//コード：
//```php
<?php
    $due = intval(fgets(STDIN));
    if ($due<=3) {
        echo "BOMB\n";
        exit(0);
    }
    echo "ONGOING\n";
?>
//```

File: ppg_sl/java/enjou.java
//言語：Java
//問題URL：https://paiza.jp/works/mondai/ppg_sl_java/ppg_sl_java__enjou
//コメント：
//コード：
//```Java

import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int due = scanner.nextInt();
        scanner.close();

        if (due<=3) {
            System.out.println("BOMB");
            System.exit(0);
        }

        System.out.println("ONGOING");
    }
}
//```